### ISSUE TITLE
integration tests broken

### ISSUE THREAD

**Issue by g-k** [mozilla-services/autograph#468]
All CI jobs time out waiting for autograph-app-hsm to come up.

Locally it looks like the keys aren't being generated or a config value is missing:

```found suitable end-entity labeled \"\" in database but not in hsm```

```console
autograph-app-hsm | {"Timestamp":1586208091467833315,"Time":"2020-04-06T21:21:31Z","Type":"app.log","Logger":"autograph","Hostname":"b901ddd692ec","EnvVersion":"2.0","Pid":1,"Severity":2,"Fields":{"msg":"failed to add signer \"normandy\": contentsignaturepki \"normandy\": failed to initialize end-entity: contentsignaturepki \"normandy\": failed to find suitable end-entity: found suitable end-entity labeled \"\" in database but not in hsm: crypto11: could not find PKCS#11 key"}}
```

**Comment by g-k** [comment]
The last successful job in CI was https://app.circleci.com/pipelines/github/mozilla-services/autograph/jobs/4264 for https://github.com/mozilla-services/autograph/pull/463 which landed in https://github.com/mozilla-services/autograph/commit/0efaaed

**Comment by g-k** [comment]
fixed in #469
